Source/WebCore/platform/RectEdges.h

```diff
@@ -73,6 +73,21 @@ template<typename T> class RectEdges {
     void setBottom(const T& bottom) { setAt(BoxSide::Bottom, bottom); }
     void setLeft(const T& left) { setAt(BoxSide::Left, left); }
 
+    RectEdges<T> flipX()
```

Comment:
These functions should be `const`. Try to use naming that helps communicate that they don't mutate the object, but return a copy, like `edgesFlippedInX()` or `xFlippedEdges()` something (IntRect has some examples of this kind of naming).

Reply:
Went with `xFlippedCopy` because there are various derivative classes only some of which use "edges" as their identifying noun...